### DISCUSSION TITLE
Slash in Containername

### DISCUSSION THREAD

**Discussion by newhinton** [containrrr/watchtower#1775] [Q&A]
Hello! How do i remove the slash in the container name from my notification?

I use the following template:
```go
        {{- if .Report -}}
          {{- with .Report -}}
        Those Containers can be updated:
             {{- range .Stale}}
        {{.Name}} ({{.ImageName}})
            {{- end -}}
          {{- end -}}
        {{- end -}}
```

though it will always print: 

Those containers can be updated:
/service.name (container:latest)

How do i get rid of it, and why is it there in the first place?

**Comment by piksel** [comment]
To remove it you can use:
```go
{{ slice .Name 1 }}
```

As to why it's there in the first place, we are just forwarding the Name field of the container inspect:
```
$ docker inspect watchtower | jq '.[].Name'
"/watchtower"
```
My guess is that it's supposed to be `host/name`, where an empty host means the local machine, but that's just a guess. We could of course always remove the initial slash, as that is what most docker tools seem to do, but we are generally reluctant to change these things, as they might break peoples setups.
